composables/task/tokioTask.ts

```diff
@@ -144,6 +144,20 @@ export class TokioTask {
 
         return TaskState.Idle;
     }
+
+    wakerCount(): bigint {
+        return this.stats.wakerClones - this.stats.wakerDrops > 0n
+            ? this.stats.wakerClones - this.stats.wakerDrops
+            : 0n;
+    }
+
+    lastWakeDuration(): Duration | undefined {
```

Comment:
How about using null? The difference between undefined and null is significant.

Reply:
Updated. Thanks!